systemvm/debian/opt/cloud/bin/cs/CsNetfilter.py

```diff
@@ -339,6 +339,9 @@ def to_str(self, delete=False):
                 printable = printable.replace('!_-', '! -')
                 if delete:
                     printable = printable.replace('-A', '-D')
+                    if k == "--save-mark":
```

Comment:
@RodrigoDLopez 
what's the purpose of this change ?

Reply:
Hi Wei,

I talked with Rodrigo, and he said that this change was made to fix an error when deleting port forwarding rules where the VR couldn't find the correct rule to delete. We couldn't figure out the exact situation that resulted in this error though.

From what I've tested so far, this change does not seem necessary (at least anymore). Maybe the problem that caused the error got fixed in another PR. Therefore, I'm going to remove these lines, finish adjusting this PR and do some extensive testing. If it really does not seem to affect anything, I'll leave the PR without these changes.